Assets/BossRoom/Scripts/Server/Game/Character/ServerCharacterMovement.cs

```diff
@@ -44,6 +44,12 @@ public class ServerCharacterMovement : NetworkBehaviour
         // this one is specific to knockback mode
         private Vector3 m_KnockbackVector;
 
+#if UNITY_EDITOR || DEVELOPMENT_BUILD
+        [SerializeField]
+        float m_CheatSpeed = 20;
```

Comment:
Once this value is value is agreed upon, this speed should just be stored as a const.